src/test/java/com/cloudbees/jenkins/support/configfiles/OtherConfigFilesComponentTest.java

```diff
@@ -88,7 +90,7 @@ public void shouldPutAPlaceHolderInsteadOfSecret() throws Exception {
         File file = File.createTempFile("test", ".xml");
         FileUtils.writeStringToFile(file, xml);
         String patchedXml = SecretHandler.findSecrets(file);
-        assertEquals(expectedXml, patchedXml);
+        assertThat(patchedXml, equalToIgnoringWhiteSpace(expectedXml));
```

Comment:
Slightly worried by that though. Is this because there is a path stored in the XML?